### PR TITLE
[DEV APPROVED] - 9111 clear filters

### PR DESCRIPTION
**Pull request by margOnline**
[TP 9111](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=board/5580782241563718174&appConfig=eyJhY2lkIjoiRTQ4MDEyNENDOUYxRDU4RDg0RTEyRjNCODBEN0VBMUYifQ==&searchPopup=userstory/8774), which is a task for 8774, Evidence Hub search filtering.

### CODE REVIEW

**Comment by margOnline** [comment]
There is an issue with the vcr cassettes whereby the file name is too long if I test with more than one filter applied. As a short term solution, I've only selected one filter to ensure that the `clear filter` feature is tested.

Suggestions re addressing this issue? I started to look at [vcr assistant](https://github.com/pat/vcr_assistant), but perhaps one of you has a better idea?

Also now would be a good time for a review generally on this pr.

File: features/step_definitions/evidence_hub/search_steps.rb [hunk 1]
```diff
@@ -2,11 +2,24 @@
   evidence_summaries_page.load(locale: 'en')
 end
 
-When('I search on evidence hub the keyword {string}') do |keyword|
+Given("I search based on some filters") do
```
Comment (willhall88) [inline features/step_definitions/evidence_hub/search_steps.rb]: I know we don't technically cover rubocop over features, but I think we should stick to the convention of single quotes on steps 
`Given('I search based on some filters') do`

**Comment by willhall88** [comment]
#90 should help with the issue of long VCR filenames

**Comment by margOnline** [comment]
@willhall88 re the request spec, yes, I didn't think it was a very good test and the feature spec is better.

**Comment by willhall88** [comment]
#90 is pointing to this branch. would you like me to merge it or would you prefer to do so? @margOnline

**Pull request merged**